debezium-testing/pom.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
```

Comment:
This module should be added to the Travis config file side-by-side with outbox

Reply:
Done.